src/fireblocks-sdk.ts

```diff
@@ -1150,4 +1153,47 @@ export class FireblocksSDK {
         }
         return await this.apiClient.issueGetRequest(url);
     }
+
+    /**
+     *
+     * @param tokenId
+     */
+    public async getNFT(tokenId: string): Promise<Token> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/tokens/${tokenId}`);
+    }
+
+    /**
+     *
+     * @param tokenIds comma seperated list of token ids to fetch
+     */
+    public async getNFTs(tokenIds: string): Promise<Token[]> {
```

Comment:
WDYT for having array of string here?

```suggestion
    public async getNFTs(tokenIds: string[]): Promise<Token[]> {
```